src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -23,8 +23,7 @@
 import hudson.plugins.git.IGitAPI;
 import hudson.plugins.git.IndexEntry;
 import hudson.plugins.git.Revision;
-import hudson.util.ArgumentListBuilder;
-import hudson.util.Secret;
+import hudson.util.*;
```

Comment:
Wildcard imports are generally discouraged, particularly for a package as broad as hudson.util.  I don't see any change that makes this change compelling.

Help me understand why it was switched to a wildcard import?

Reply:
I also dislike wildcard imports.  As someone who regularly digs through source code wildcard imports make any code digging expensive time-wise if there’s many wildcard imports and no known package without digging.

Reply:
I'm using IntelliJ, and the default setting is to use a wildcard after >=5 imports from the same namespace have been used. So this refactoring happened automatically (though I did stage/commit it manually, of course). I did a quick code grep for `.*;` and found several other instances in the codebase where wildcard imports were being used, so I thought it would be ok.

Anyways, I don't have a strong opinion one way or another on the matter, I'd be happy to remove the wildcard and explicitly list the imports. I'll push a fixup for this.